distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -270,6 +270,12 @@ public String toString() {
 
     public static final DynamicPort RMI_REG_PORT = new DynamicPort(5);
 
+    public static final DynamicPort AMQP_PORT = new DynamicPort(101);
```

Comment:
Probably better to go back to 6/7/8 since that's what other test classes with dynamic ports do.